### ISSUE TITLE
[Bug]: fire doesnt burn you it seem. only the overheating from being set on fire

### ISSUE THREAD

**Issue by kiekatz** [BeeStation/BeeStation-Hornet#11157]
### BYOND Version

515

### Round Date

N/A

### Round ID

N/A

### Test merges

N/A

### Describe the bug

When I [set myself on fire] as you do, I noticed [that it takes some time for my character to start actually taking damage.]

upon testing it took ~9s for any damage to start to build up. from splashing with 100u of wielding fuel and getting hit with a lighter.
only getting to a moderate burn damage overtime from the overheating

considering there are several races who have parts of their positives and negatives relating around fire resistance/weakness this bug/stealth nerf has most likely minorly shifted the balance of races

### What did you expect?

I expected [fire to hurt]

### Reproducible steps

1. set yourself on fire
2. count around 9 seconds before you take any damage. funnily enough its usually by the time your clothes also burn up

**Comment by zeskorion** [comment]
this has always been the case

**Comment by kiekatz** [comment]
i strongly doubt this has "always been the case" dont tell me im delusional

**Comment by zeskorion** [comment]
> i strongly doubt this has "always been the case" dont tell me im delusional
> 
> im fairly certain i would remember "back in the day" fire taking around 9s to actually hurt

nope. You usually just encountered fire with circumstances that damaged you instantly (lava, explosions), or heated you up way faster (plasmafire). Check the git logs for firestacks, nothing's changed as damage is concerned.
